src/App.tsx

```diff
@@ -21,42 +24,75 @@ const routes = [
             </RootLayout>
         ),
     },
+
     {
         path: '/template_editor',
         element: <TemplateEditor />,
     },
-].concat(
-    Object.keys(templatesConfig).flatMap(dbName => [
-        {
-            path: `/app/${dbName}`,
-            // TODO: Create a component that provides the functionality
-            // to manage the documents in this DB
-            element: (
-                <RootLayout>
-                    <div>
-                        <JobsView dbName={dbName} />
-                    </div>
-                </RootLayout>
-            ),
-        },
-        {
-            path: `/app/${dbName}/:docId`,
-            element: (
-                <RootLayout>
-                    <MdxTemplateView dbName={dbName} />
-                </RootLayout>
-            ),
-        },
-        {
```

Comment:
Please keep this route, but comment it out and leave a comment to say this does not work due to recent changes and we should return to this later.

Reply:
Added proper comments in App.tsx for each route.